YoCode/Directory.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace YoCode
+{
+    public class Directory : IDirectory
+    {
+        public List<String> OriginalPaths { get; }
+        public List<String> ModifiedPaths { get; }
+
+        public Directory(List<String> originalPaths, List<String> modifiedPaths)
+        {
+            OriginalPaths = originalPaths;
+            ModifiedPaths = modifiedPaths;
+        }
+        private List<Stream> ReturnPathFileStream(List<string> paths)
+        {
+            List<Stream> streamList = new List<Stream>();
+
+            for (int i = 0; i < paths.Count; i++)
+            {
+                FileStream fs = File.Create(paths[i]);
+                streamList.Add(fs);
+            }
+            return streamList;
+        }
+        public List<Stream> ReturnOriginalPathFileStream()
+        {
+            return ReturnPathFileStream(OriginalPaths);
+        }
+        public List<Stream> ReturnModifiedPathFileStream()
+        {
+            return ReturnPathFileStream(ModifiedPaths);
```

Comment:
Could have one method  ReturnPathFileStream() ?

Reply:
We made these two methods when we were writing the unit tests. You're right that we could have one, and it used to be one, but making 2 made it easier to write the unit tests.